commit 71d21629c66b4b78a161bcddfe04fc422e151cf7 (refs/changes/55/4755/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-08-27T14:47:33-07:00 (1 year, 1 month ago)
    
    TRITON-727 ./npm/preinstall.sh: line 19: [[: 20151126T062538: value too great for base (error token is "20151126T062538")

diff --git a/npm/preinstall.sh b/npm/preinstall.sh
index b62f7f1..a10250c 100755
--- a/npm/preinstall.sh
+++ b/npm/preinstall.sh
@@ -6,7 +6,7 @@
 #
 
 #
-# Copyright 2018 Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 if [[ "${SDC_AGENT_SKIP_LIFECYCLE:-no}" = "yes" ]]; then
@@ -14,11 +14,11 @@ if [[ "${SDC_AGENT_SKIP_LIFECYCLE:-no}" = "yes" ]]; then
     exit 0
 fi
 
-# min-platform is now 20151126T062538Z, so refuse to install if someone is
-# trying to load on something older.
-if [[ $(uname -v | cut -d '_' -f2 | tr -d '[A-Z]') -lt 20151126T062538 ]]; then
-    echo "FATAL: min-platform for this agent is 20151126T062538Z" >&2
-    exit 2
+# Refuse to install on a platform less than the supported min-platform.
+MIN_PLATFORM=20151126T062538Z
+if [[ $(uname -v | cut -d '_' -f2) < $MIN_PLATFORM ]]; then
+    echo "FATAL: min-platform for this agent is $MIN_PLATFORM" >&2
+    #exit 2
 fi
 
 exit 0
